2차/윤형진_소수-찾기.py

```diff
@@ -0,0 +1,21 @@
+def solution(n):
+    answer = 0
+    
+    if n == 2:
+        answer = 1
+    else :
+        answer = 1
+        for i in range(3, n + 1) :
+            is_prime_num = True
+            for j in range(2, i) :
+                if i%j == 0 :
+                    is_prime_num = False
+                    break
+                    
+            if is_prime_num == True :
+                answer = answer + 1
```

Comment:
이 부분은 구조 분해를 하는게 좋을 것 같아요

```suggestion
    if n == 2:
        return 1
    answer = 1
    for i in range(3, n + 1) :
        is_prime_num = True
        for j in range(2, i) :
            if i%j == 0 :
                is_prime_num = False
                break
                    
        if is_prime_num == True :
            answer = answer + 1
```